unittests/snapshots/CMakeLists.txt

```diff
@@ -23,3 +23,7 @@ configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/snap_v6.bin.json.gz ${CMAKE_CURRENT_
 configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/snap_v7.bin.gz ${CMAKE_CURRENT_BINARY_DIR}/snap_v7.bin.gz COPYONLY )
 configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/snap_v7.json.gz  ${CMAKE_CURRENT_BINARY_DIR}/snap_v7.json.gz  COPYONLY )
 configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/snap_v7.bin.json.gz ${CMAKE_CURRENT_BINARY_DIR}/snap_v7.bin.json.gz COPYONLY )
```

Comment:
Should we get rid of the `snap_v7` files now?

Reply:
https://github.com/AntelopeIO/spring/pull/719/commits/1bdd27645b5ab83eaa630970d9ccc55e7241d2f8